Exercises/reverse-string.js

```diff
@@ -0,0 +1,33 @@
+// Using the JavaScript language, have the function FirstReverse(str) take the str parameter
+// being passed and return the string in reversed order.
+
+// Input:"coderbyte"
+// Output:"etybredoc"
+//
+// Input:"I Love Code"
+// Output:"edoC evoL I"
+
+//Method 1
+function FirstReverse(str){
+    var stringArray = [];
+    reverseArray = [];
+
+    stringArray = str.split("");
+
+    for(i=stringArray.length-1; i>=0; i--){
+      reverseArray.push(stringArray[i]);
+    }
+
+    str = reverseArray.join("");
+    return str
+}
+
+FirstReverse(readline());
+
+//Method 2
+function FirstReverse(str){
```

Comment:
Since these are in the same file, I'd lean towards naming the second version differently than the first.